swf/src/types.rs

```diff
@@ -387,6 +387,8 @@ pub enum BlendMode {
     Erase = 12,
     Overlay = 13,
     HardLight = 14,
+    // FIXME - prevent this from being read from SWF
+    Shader = 15,
```

Comment:
For cases like these, I think we should have a similar (but "enhanced") enum in ruffle_render, which has the new types that a swf can't know about. It feels like a big hack to have these exist in a serialization crate.